[BUG] Impossible to compile when LEVELING_NOZZLE_TEMP is disabled

Impossible to compile when #define LEVELING_NOZZLE_TEMP is disabled.

#define PREHEAT_BEFORE_LEVELING
#if ENABLED(PREHEAT_BEFORE_LEVELING)
  //#define LEVELING_NOZZLE_TEMP 120   // (°C) Only applies to E0 at this time
  #define LEVELING_BED_TEMP     60
#endif


VsCode return : 
'LEVELING_NOZZLE_TEMP' was not declared in this scope; did you mean 'LEVELING_BED_TEMP'?

```
Marlin/src/gcode/bedlevel/abl/G29.cpp: In static member function 'static void GcodeSuite::G29()':
Marlin/src/gcode/bedlevel/abl/G29.cpp:413:48: error: 'LEVELING_NOZZLE_TEMP' was not declared in this scope; did you mean 'LEVELING_BED_TEMP'?
  413 |         if (!dryrun) probe.preheat_for_probing(LEVELING_NOZZLE_TEMP, LEVELING_BED_TEMP);
      |                                                ^~~~~~~~~~~~~~~~~~~~
      |                                                LEVELING_BED_TEMP
```

Tested on latest bugfix. It worked a few builds before.

--- a/Marlin/src/module/probe.cpp
+++ b/Marlin/src/module/probe.cpp
@@ -335,14 +335,6 @@ FORCE_INLINE void probe_specific_action(const bool deploy) {
       #define PROBING_BED_TEMP 0
     #endif
   #endif
-  #if ENABLED(PREHEAT_BEFORE_LEVELING)
-    #ifndef LEVELING_NOZZLE_TEMP
-      #define LEVELING_NOZZLE_TEMP 0
-    #endif
-    #ifndef LEVELING_BED_TEMP
-      #define LEVELING_BED_TEMP 0
-    #endif
-  #endif
 
   /**
    * Do preheating as required before leveling or probing.

--- a/Marlin/src/module/probe.h
+++ b/Marlin/src/module/probe.h
@@ -44,6 +44,15 @@
   #define PROBE_TRIGGERED() (READ(Z_MIN_PIN) != Z_MIN_ENDSTOP_INVERTING)
 #endif
 
+#if ENABLED(PREHEAT_BEFORE_LEVELING)
+  #ifndef LEVELING_NOZZLE_TEMP
+    #define LEVELING_NOZZLE_TEMP 0
+  #endif
+  #ifndef LEVELING_BED_TEMP
+    #define LEVELING_BED_TEMP 0
+  #endif
+#endif
+
 class Probe {
 public:
 

